docs/man/tldr.1

```diff
@@ -0,0 +1,94 @@
+.\" Man page generated from reStructuredText.
```

Comment:
Is there value in having this generated file as part of the checked-in source code? I feel like better to have this be generated on demand for when we'll need it, like in the [publish](https://github.com/tldr-pages/tldr-python-client/blob/main/.github/workflows/publish.yml) workflow.

Reply:
absolutely, your are right! I'm gonna adjust the git action file to auto-generate it